### ISSUE TITLE
Add support for VNET integration

### ISSUE THREAD

**Issue by spoplavskiy** [jenkinsci/azure-container-agents-plugin#48]
In mid November 2019 Microsoft announced a Linux preview support of custom virtual networks on Azure Container Instances: [details here](https://azure.microsoft.com/en-us/updates/aci-vnet/)
This is very useful feature, that allows customers to hide and secure their build environment from external world.

**Comment by jplorier** [comment]
Hi, I've created a new issue #59 to push the code for this feature, still waiting for a reply on the emails to the head developers.

**Comment by cutty201** [comment]
Yes! We need this --- I am about to look at adding it myself I am going to yammer about this to our Cloud Solutions Architects
